packages/core/utils/src/sourcemap.js

```diff
@@ -30,15 +30,18 @@ export async function loadSourceMapUrl(
   if (match) {
     let url = match[1].trim();
     let dataURLMatch = url.match(DATA_URL_RE);
-    filename = dataURLMatch ? filename : path.join(path.dirname(filename), url);
+    let mapFilePath = dataURLMatch ? filename : url.replace(/^file:\/\//, '');
+    if (!dataURLMatch && !isAbsolute(mapFilePath)) {
+      mapFilePath = path.join(path.dirname(filename), mapFilePath);
+    }
```

Comment:
```suggestion
    let mapFilePath;
    if (dataURLMatch) {
        mapFilePath =  isAbsolute(mapFilePath) ? filename : path.join(path.dirname(filename), mapFilePath);
    } else {
        mapFilePath = url.replace(/^file:\/\//, '');
    }
```

Reply:
Will revert this change, it seems incorrect

Reply:
```suggestion
    let mapFilePath;
    if(dataURLMatch) {
        mapFilePath = filename;
    } else {
        mapFilePath = isAbsolute(mapFilePath) ? url.replace(/^file:\/\//, '') : path.join(path.dirname(filename), mapFilePath);
    }
```